README.md

```diff
@@ -1 +1,46 @@
-# Ruby-project1
\ No newline at end of file
+# Capstone Project NTML & CSS
+
+This is the Ruby project #1, from the Microverse's main curriculum and aims to build two algorithms similar to the sorting method, without using those methods.
+
+![screenshot](./assets/Ruby-pro_1.png)
+
+## Built With
+
+- RUBY
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
```

Comment:
You are instructing the user to follow the steps to get a local copy. But there are no steps written.

- [ ] Please add the steps to set up the project locally.